model/Deserialization/Classes/Deserialization.md

```diff
@@ -0,0 +1,36 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Deserialization
+
+## Summary
+
+An element that can store information about a deserialization event, minted on the fly and not persisted in the payload.
+
+## Description
+
+tbd.
+
+Propreties should be used as follows:
+
+- element, to encode the content that was generated by when parsing the file
```

Comment:
I'm a bit confused about the element property description - would this reference that actual elements in the graph deserialized from the file?

Reply:
Makes sense - I may suggest some word smithing if we go down this route.